utils/io/fileutils/files.go

```
@@ -571,20 +571,29 @@ func MoveDir(fromPath, toPath string) error {
 // MoveFile(source, destination) will work moving file between folders
 // Therefore, we are using our own implementation (MoveFile) in order to rename files.
 func MoveFile(sourcePath, destPath string) error {
+	inputFileOpen := true
 	inputFile, err := os.Open(sourcePath)
 	if err != nil {
 		return errorutils.CheckError(err)
 	}
-	defer inputFile.Close()
+	defer func() {
+		if inputFileOpen {
+			inputFile.Close()
```

The returned error is ignored